MineLeapMod/src/com/nautigsam/mineleapmod/GameRenderHandler.java

```diff
@@ -177,46 +173,52 @@ private static void DrawRetical()
 
 	private static void UpdateInGameCamera()
 	{
-		if (mc.thePlayer != null)
+		if (mc.thePlayer != null && ControllerSettings.isLeapMotionEnabled())
 		{
+			LeapMotionMouse lmm = ControllerSettings.getLeapMotionMouse();
 			if (lastFlansModCheckValue)
 			{
-				if (LeapMotionMouse.pollAxis(false))
+				if (lmm.pollAxis(false))
 				{
 					float multiplier = 4f * mc.gameSettings.mouseSensitivity;
 					VirtualMouse.moveMouse(
-							(int) (LeapMotionMouse.deltaX * multiplier),
-							(int) (LeapMotionMouse.deltaY * multiplier));
+							(int) (lmm.deltaX * multiplier),
+							(int) (lmm.deltaY * multiplier));
 				}
 				else
 				{
 					VirtualMouse.moveMouse(0, 0);
 				}
 			}
-			else if (LeapMotionMouse.pollAxis(false))
+			else if (lmm.pollAxis(false))
 			{
-				mc.thePlayer.setAngles(LeapMotionMouse.deltaX, LeapMotionMouse.deltaY);
+				mc.thePlayer.setAngles(lmm.deltaX, lmm.deltaY);
 			}
 		}
 	}
 
 	private static void HandleDragAndScrolling()
 	{
-
-		if (VirtualMouse.isButtonDown(0) || VirtualMouse.isButtonDown(1))
-		{
-			// VirtualMouse.moveMouse(LeapMotionMouse.getmcX(), LeapMotionMouse.getmcY());
-			McGuiHelper.guiMouseDrag(LeapMotionMouse.getX(), LeapMotionMouse.getY());
-			VirtualMouse.setMouseButton(LeapMotionMouse.isLeftButtonDown() ? 0 : 1, true);
-		}
+//		if (ControllerSettings.isLeapMotionEnabled())
+//		{
+//			LeapMotionMouse lmm = ControllerSettings.getLeapMotionMouse();
+//			if (VirtualMouse.isButtonDown(0) || VirtualMouse.isButtonDown(1))
+//			{
+//				// VirtualMouse.moveMouse(LeapMotionMouse.getmcX(), LeapMotionMouse.getmcY());
+//				McGuiHelper.guiMouseDrag(lmm.getX(), lmm.getY());
+//				VirtualMouse.setMouseButton(lmm.isLeftButtonDown() ? 0 : 1, true);
+//			}
+//		}

```

Comment:
I don't remember why it didn't work at some point, I will try to uncomment it later.